app/src/main/java/com/globant/example/mentorapp/presentation/presenter/ListUsersPresenterImpl.java

```diff
@@ -0,0 +1,62 @@
+package com.globant.example.mentorapp.presentation.presenter;
+
+import android.support.annotation.NonNull;
+
+import com.globant.example.mentorapp.data.entity.EventApiResponseEntity;
+import com.globant.example.mentorapp.domain.interactor.ListUsersInteractorImpl;
+import com.globant.example.mentorapp.presentation.di.Component.ApplicationComponent;
+import com.globant.example.mentorapp.presentation.model.SharedUserViewModel;
+import com.globant.example.mentorapp.presentation.view.fragment.ListUsersFragment;
+import com.squareup.otto.Bus;
+import com.squareup.otto.Subscribe;
+
+import javax.inject.Inject;
+
+/**
+ * This class controls the exchange of information between the view and the data sources
+ * Created by oscar.amezquita on 7/06/2017.
+ */
+
+public class ListUsersPresenterImpl implements ListUsersPresenter {
+
+    @Inject
+    ListUsersInteractorImpl interactor;
```

Comment:
If you're using interfaces to define views, presenters and interactors, each component should talk with the interface object, not the implementation object.